### DISCUSSION TITLE
[Feature] Add Dog face to be named and searched on.

### DISCUSSION THREAD

**Discussion by mckayit** [immich-app/immich#14959] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great to be able to search for a Dog and name him   so you can then find all pictures of that Dog.    
If i search for Dog   i get to see all pictures with Dog   but it would be a great add on to the AI to also name the Dog and then have the option to search by name.

![459508260_122177000012188324_8640595207631977986_n](https://github.com/user-attachments/assets/a2daea79-8703-49af-ac51-89e6134a2a6b)

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mmomjian** [comment]
thank you, duplicate of #7151
